dsdl_compiler/libcanard_dsdl_compiler/code_type_template.tmpl

```diff
@@ -29,7 +40,7 @@
   * @param root_item: for detecting if TAO should be used
   * @retval returns offset
   */
-uint32_t ${type_name}_encode_internal(${type_name}* source, void* msg_buf, uint32_t offset, uint8_t root_item)
+uint32_t ${type_name}_encode_internal(${type_name}* source, void* msg_buf, uint32_t offset, uint8_t CANARD_MAYBE_UNUSED(root_item))
```

Comment:
This line and similar definitions below are too long. Please reformat the code to make each line no longer than 120 characters. I realize it's not your fault but why not fix it while we're at it.